src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/ScannableItemRepository.java

```diff
@@ -1,14 +1,9 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.entity;
 
 import org.springframework.data.jpa.repository.JpaRepository;
-import org.springframework.data.jpa.repository.Modifying;
-import org.springframework.data.jpa.repository.Query;
-import org.springframework.data.repository.query.Param;
 
 import java.util.UUID;
 
 public interface ScannableItemRepository extends JpaRepository<ScannableItem, UUID> {
```

Comment:
can we remove this repository entirely?

Reply:
This repository is used in DocumentProcessor. I'm removing a method I've recently added, because it won't be used.